src/exec/storage.go

```diff
@@ -51,6 +51,16 @@ func (s storage) Run(config config.Config) bool {
 		return false
 	}
 
+	if err := s.createPasswd(config); err != nil {
+		s.Logger.Crit("failed to create users/groups: %v", err)
+		return false
+	}
+
+	if err := s.createFilesystemsFiles(config); err != nil {
```

Comment:
Doesn't this replace the existing method for writing files? I don't see any deleted lines in this PR.